indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -139,31 +170,9 @@
                 </div>
             {% endfor %}
         </div>
-        <div class="i-box-footer">
-            {% set url = url_for('.judge_abstract', abstract) %}
-            {% set form_id = 'judgment-form' %}
-
-            {% call form_header(form, action=url, orientation='vertical', id=form_id) %}
-                data-form-container="#abstract-decision-box"
-                data-update="#abstract-page"
-                data-ajax-form
-            {% endcall %}
-            {% call(link, endlink) render_instruction_box(form_id, event.cfa.judgment_instructions, _("Judgment instructions")) %}
-                {% trans -%}
-                    Please don't forget to read the {{ link }}judgment instructions{{ endlink }} before taking your decision.
-                {%- endtrans %}
-            {% endcall %}
-            {{ form_rows(form, fields=form._order, skip_labels=true) }}
-            {% call form_footer(form) %}
-                <button class="i-button big highlight">
-                    {% trans %}Judge{% endtrans %}
-                </button>
-            {% endcall %}
-        </div>
     </div>
 {% endmacro %}
 
-
```

Comment:
This is starting to become another thing to write an entry in the wiki. Are we adding two blank lines before macros and blocks or not? I think having two makes it slightly clearer.